app/src/i18n/supportedLocales.js

```diff
@@ -37,9 +37,18 @@ export default {
   eu: {
     name: 'Euskara'
   },
+  fa: {
+    name: 'فارسی'
+  },
```

Comment:
```suggestion
  fa: {
    name: 'فارسی',
    dateFnsLocale: 'fa-IR'
  },
```
Looks like data-fns (lib we're using for i18n dates) doesn't implement the global Persian `fa` locale but the local Iranian Persian `fa-IR` [is present](https://github.com/date-fns/date-fns/blob/master/src/locale/fa-IR/index.js), maybe it will be a better fallback than english ?